pkg/project/project.go

```diff
@@ -5,7 +5,7 @@ var (
 	gitSHA             = "n/a"
 	name        string = "aws-operator"
 	source      string = "https://github.com/giantswarm/aws-operator"
-	version            = "10.0.1-dev"
+	version            = "10.1.1"
```

Comment:
That looks wrong. Only the Github Action for releases should change the version like this. If you want to test you can customize the suffix to your liking.